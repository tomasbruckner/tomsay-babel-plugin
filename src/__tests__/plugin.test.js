const babel = require('@babel/core');
const plugin = require('../plugin');

var example = `
console.tom('XD');
`;

it('Basic example', () => {
  const { code } = babel.transform(example, { plugins: [plugin] });
  expect(code).toMatchSnapshot();
});
