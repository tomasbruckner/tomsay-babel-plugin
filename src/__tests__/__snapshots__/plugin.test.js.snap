// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic example 1`] = `
"console.log(\\"----------------\\");
console.log('XD');
console.log(\\"----------------\\");
console.log(\\" \\\\n .----------------.  .----------------.  .----------------.   .----------------.  .----------------.  .----------------.  .----------------.  \\\\n| .--------------. || .--------------. || .--------------. | | .--------------. || .--------------. || .--------------. || .--------------. |\\\\n| |  _________   | || |     ____     | || | ____    ____ | | | |    _______   | || |      __      | || |  ____  ____  | || |    _______   | |\\\\n| | |  _   _  |  | || |   .'    \`.   | || ||_   \\\\\\\\  /   _|| | | |   /  ___  |  | || |     /  \\\\\\\\     | || | |_  _||_  _| | || |   /  ___  |  | |\\\\n| | |_/ | | \\\\\\\\_|  | || |  /  .--.  \\\\\\\\  | || |  |   \\\\\\\\/   |  | | | |  |  (__ \\\\\\\\_|  | || |    / /\\\\\\\\ \\\\\\\\    | || |   \\\\\\\\ \\\\\\\\  / /   | || |  |  (__ \\\\\\\\_|  | |\\\\n| |     | |      | || |  | |    | |  | || |  | |\\\\\\\\  /| |  | | | |   '.___\`-.   | || |   / ____ \\\\\\\\   | || |    \\\\\\\\ \\\\\\\\/ /    | || |   '.___\`-.   | |\\\\n| |    _| |_     | || |  \\\\\\\\  \`--'  /  | || | _| |_\\\\\\\\/_| |_ | | | |  |\`\\\\\\\\____) |  | || | _/ /    \\\\\\\\ \\\\\\\\_ | || |    _|  |_    | || |  |\`\\\\\\\\____) |  | |\\\\n| |   |_____|    | || |   \`.____.'   | || ||_____||_____|| | | |  |_______.'  | || ||____|  |____|| || |   |______|   | || |  |_______.'  | |\\\\n| |              | || |              | || |              | | | |              | || |              | || |              | || |              | |\\\\n| '--------------' || '--------------' || '--------------' | | '--------------' || '--------------' || '--------------' || '--------------' |\\\\n '----------------'  '----------------'  '----------------'   '----------------'  '----------------'  '----------------'  '----------------' \\");"
`;
